### ISSUE TITLE
Setting stack limit does not work in OS X

### ISSUE THREAD

**Issue by torb** [Kattis/problemtools#42]
This line in program.py raises an error, even when running as sudo:
resource.setrlimit(resource.RLIMIT_STACK, (resource.RLIM_INFINITY, resource.RLIM_INFINITY))

After IDI open this weekend, I decided to use your problemtools for the entire process with future contests. I got almost everything working on OS X, but this line causes a crash. I have not yet found a way around it.

This only affects running python solutions. If I remove this line, I am able to verify solutions in other languages, but all python processes crash.

I know you currently don't support OS X, but I thought I'd let you know anyway. I will create a pull request if I am able to fix it.

**Comment by torb** [comment]
Another issue was that I did not have /usr/bin/python2
Commenting out this line and fixing the python2 issue allowed me to run verifyproblem on OS X
So I am able to use your tools on OS X, but only after making some changes (hacks)

**Comment by austrin** [comment]
@torb does this work better on the current version of problemtools?  If not, can you post a stack trace of the crash you get?

**Comment by austrin** [comment]
@torb I'll close this since I believe this may be fixed (but don't have a Mac so can't test).  If this is still an issue feel free to reopen this, and please provide terminal output including stack trace.
